devcluster/example.yaml

```diff
@@ -108,6 +108,9 @@ stages:
         # This is important: we have to use the symbolic links in the
         # tools/build directory to run properly.
         root: tools/build
+        # A dedicated directory for caching file. This directory is assumed to be empty.
+        cache:
+          cache_dir: /var/cache/determined
```

Comment:
shouldn't this be somewhere user-writable by default?

Reply:
Like `/tmp/determined-cache`? 
I was debating about this... because if we have `/tmp/determined-cache` as default, we are putting cached files into user's `/tmp` directory (which we are not suppose to) without prior notice... choosing an access denied folder (but designed for cache) would let user either grant access, or choose another folder

Reply:
You can put junk into your **coworker's** `/tmp`, but not into your **customer's** `/tmp`.

`devcluster` is just for us devs, basically, and we want for it to be easy for new devs to get started.

Reply:
I see, yea I think this makes sense, this has been fixed.